frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx

```diff
@@ -57,6 +64,7 @@ export function DestinationsFilters({
                         onChange={(e) => setFilters({ kind: e ?? null })}
                     />
                 )}
+                {askForFeedback ? null : <NewButton stage={PipelineStage.Destination} size="small" />}
```

Comment:
Not a fan of this. Why does askForFeedback determine if there is a new button? 

Either this should have its own prop or it should be just done outside of this component.

Reply:
added a `hideAddDestinationButton` property for the new button